src/Standards/PSR2/Sniffs/Classes/ClassDeclarationSniff.php

```diff
@@ -231,10 +248,11 @@ public function processOpen(File $phpcsFile, $stackPtr)
         $classCount         = count($classNames);
         $checkingImplements = false;
         $implementsToken    = null;
-        foreach ($classNames as $i => $className) {
+        foreach ($classNames as $n => $className) {
```

Comment:
The change here is not because of my preference, but `$i` is used in another loop inside and because of that in some case we had invalid check in line 398.